client_end_user/shared/src/commonMain/kotlin/di/screenModelsModule.kt

```diff
@@ -27,5 +26,6 @@ val screenModelsModule = module {
     factoryOf(::PickLanguageScreenModel)
     factoryOf(::PreferredRideScreenModel)
     factoryOf(::NotificationScreenModel)
+    factoryOf(::CuisinesScreenModel)
     factory { AppScreenModel(get()) }
```

Comment:
why this model only use this approach in injection?, i think it missed, so we need to refactor it

Reply:
Sadly it's out of PR scope.. may we chance it in another PR